src/authorization/WebAclAuthorizer.ts

```diff
@@ -15,29 +15,35 @@ import { NotFoundHttpError } from '../util/errors/NotFoundHttpError';
 import { NotImplementedHttpError } from '../util/errors/NotImplementedHttpError';
 import { UnauthorizedHttpError } from '../util/errors/UnauthorizedHttpError';
 import type { IdentifierStrategy } from '../util/identifiers/IdentifierStrategy';
+import { promiseAny } from '../util/PromiseUtil';
+import { readableToQuads } from '../util/StreamUtil';
 import { ACL, FOAF } from '../util/Vocabularies';
+import type { AccessChecker } from './accessCheckers/AccessChecker';
 import type { AuthorizerArgs } from './Authorizer';
 import { Authorizer } from './Authorizer';
 import { WebAclAuthorization } from './WebAclAuthorization';
 
 /**
  * Handles most web access control predicates such as
- * `acl:mode`, `acl:agentClass`, `acl:agent`, `acl:default` and `acl:accessTo`.
- * Does not support `acl:agentGroup`, `acl:origin` and `acl:trustedApp` yet.
+ * `acl:mode`, `acl:agentClass`, `acl:agent`, `acl:default`,
+ * `acl:accessTo` and `acl:agentGroup`.
+ * Does not support `acl:origin` and `acl:trustedApp` yet.
  */
 export class WebAclAuthorizer extends Authorizer {
   protected readonly logger = getLoggerFor(this);
 
   private readonly aclStrategy: AuxiliaryIdentifierStrategy;
   private readonly resourceStore: ResourceStore;
   private readonly identifierStrategy: IdentifierStrategy;
+  private readonly accessCheckers?: AccessChecker[];
 
   public constructor(aclStrategy: AuxiliaryIdentifierStrategy, resourceStore: ResourceStore,
-    identifierStrategy: IdentifierStrategy) {
+    identifierStrategy: IdentifierStrategy, accessCheckers?: AccessChecker[]) {
```

Comment:
Should this be a single access checker, and then aggregated, @joachimvh?

Reply:
I added a prototype implementation of BooleanHandler